### DISCUSSION TITLE
integrated Terminal

### DISCUSSION THREAD

**Discussion by ECEAthipanbose** [scalacenter/bloop#2279] [Q&A]
Can bloop support integrated terminal to client(metals)?(not internal console)

**Comment by tgodzik** [comment]
That is fat out of it's scope I think. I would need to be Metals to support it, but would be great to know the use case

**Reply by ECEAthipanbose** [reply]
thank you [tgodzik](https://github.com/tgodzik) thaks for your valuable time
